cmd/snap-confine/seccomp-support.c

```diff
@@ -109,12 +136,73 @@ static void validate_bpfpath_is_safe(const char *path)
 	}
 }
 
+static void sc_cleanup_sock_fprog(struct sock_fprog *prog)
+{
+	free(prog->filter);
+	prog->filter = NULL;
+}
+
+static void sc_must_read_filter_from_file(FILE *file, uint32_t len_bytes, char *what, struct sock_fprog *prog)
+{
+	prog->len = len_bytes / sizeof(struct sock_filter);
```

Comment:
Should it be an error if `len_bytes` is not a multiple of `sizeof(struct sock_filter)`?